### PR TITLE
Add code of conduct

### PR DESCRIPTION
**Pull request by blowekamp**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+take responsibility for your impact and your mistakes – if someone says they
+have been harmed through your words or actions, listen carefully, apologize
+sincerely, and correct the behavior going forward. If someone has been harmed
+or offended, we listen carefully and respectfully, and work to right the wrong.
+
+A simple apology can go a long way. It can often de-escalate a situation, and
+telling someone that you are sorry is an act of empathy that does not
+automatically imply an admission of guilt.
+
+### Be concise
```
Comment (dave3d) [inline CODE_OF_CONDUCT.md]: Is this code of conduct **concise**?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,432 @@
+ITK Community Participation Guidelines
+======================================
+
+Version 1.0 – Updated January, 2018
+
+The [Insight Software Consortium](http://www.insightsoftwareconsortium.org/)
+(ISC) as the copyright holder of the National Library of Medicine Insight
```
Comment (dave3d) [inline CODE_OF_CONDUCT.md]: Hey look!  It's NLM!

Comment (zivy) [inline CODE_OF_CONDUCT.md]: Need to remove NLM as the toolkit was renamed.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,432 @@
...
+Conduct for SimpleITK.
+
+Purpose
+-------
+
+The primary goal of the ITK community is to be inclusive of the largest number
+of contributors, with the most varied and diverse backgrounds. Please read our
+[Motivation](Documentation/CodeOfConduct/Motivation.md) statement.
+
+Our community is build upon [values](#our-values), and we welcome and encourage
```
Comment (zivy) [inline CODE_OF_CONDUCT.md]: build -> built

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,432 @@
...
+sincerely, and correct the behavior going forward. If someone has been harmed
+or offended, we listen carefully and respectfully, and work to right the wrong.
+
+A simple apology can go a long way. It can often de-escalate a situation, and
+telling someone that you are sorry is an act of empathy that does not
+automatically imply an admission of guilt.
+
+### Be concise
+
+Value the each other's time. Keep in mind that what you write once will be read
```
Comment (zivy) [inline CODE_OF_CONDUCT.md]: Value the each other's time -> Value each other's time

**Pull request merged**